src/Components/InstanceTypes/SuggestedInstanceTypesTable.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import {
+    TableVariant,
+    TableComposable,
+    Thead, Tr, Th, Tbody, Td
+} from '@patternfly/react-table';
+import {
+    Card,
+    CardBody,
+    Pagination,
+    PaginationVariant
+} from '@patternfly/react-core';
+import { PrimaryToolbar } from '@redhat-cloud-services/frontend-components/PrimaryToolbar';
+import { PAGE, PER_PAGE } from '../../constants';
+
+const SUGG_INSTANCE_TYPES_TABLE_COLUMNS = [
+    'Suggested instance type',
+    'Provider',
+    'Description',
+    'Systems'
+];
+const SUGG_INSTANCE_TYPES_DATA = [
+    { instanceType: 't3.micro', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 1 },
+    { instanceType: 't3.small', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 1 },
+    { instanceType: 't2.small', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 2 },
+    { instanceType: 't2.micro', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 1 },
+    { instanceType: 't2.large', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 4 },
+    { instanceType: 'm2.2xlarge', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 1 },
+    { instanceType: 'm3.2xsmall', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 6 },
+    { instanceType: 'm3.2xlarge', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 1 }
+];
+
+export default function SuggestedInstanceTypesTable() {
+
+    const [page, setPage] = useState(PAGE);
+    const [perPage, setPerPage] = useState(PER_PAGE);
+
+    const onSetPage = (event, page) => setPage(page);
+    const onPerPageSelect = (event, perPage) => {
+        setPerPage(perPage);
+        setPage(PAGE);
+    };
+
+    return (
+        <>
+            <section className='pf-l-page__main-section pf-c-page__main-section'>
+                <Card>
+                    <CardBody>
+                        <PrimaryToolbar
```

Comment:
@r14chandra since we already using Pagination component from patternfly for bottom one, can we also use it for top and remove this from FEC to maintain consistency? thoughts?